ui/src/components/UCCCredit/UCCCredit.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Typography } from '@splunk/react-ui/Typography';
+import styled from 'styled-components';
+import Link from '@splunk/react-ui/Link';
+import { getUnifiedConfigs } from '../../util/util';
+
+const StyledTypography = styled(Typography)`
+    font-size: 0.8em;
+`;
+
+const UccCredit = () => {
+    const unifiedConfigs = getUnifiedConfigs();
+
+    // eslint-disable-next-line no-underscore-dangle
+    const uccVersion = unifiedConfigs?.meta?._uccVersion ?? null;
+    if (unifiedConfigs?.meta?.hideUCCVersion) {
+        return null;
+    }
```

Comment:
```suggestion
    const uccVersion = unifiedConfigs?.meta?._uccVersion;
    if (!uccVersion || unifiedConfigs?.meta?.hideUCCVersion) {
        return null;
    }
```

not sure what we should do when there is no uccVersion but maybe just do not display this info.
If we want leave it without any number just "Made with UCC", lets move that after if statement, as it is used only after it.